### ISSUE TITLE
Global_score

### ISSUE THREAD

**Issue by zjlclimb** [dauparas/ProteinMPNN#106]
Does anyone know the mathematical meaning of global_score?

**Comment by matteoferla** [comment]
As far as I can tell, average loss values of the whole protein.
there are two scores, `score` and `global_score` in the output —the codebase has a `native_score`. The scores are a negative log likelihood for the loss function (one line in sup info).
They are calculated ( https://github.com/dauparas/ProteinMPNN/blob/main/protein_mpnn_utils.py#L39) using `torch.nn.NLLLoss`. That means low is good —so like the score in a golf game not a football, cricket etc game.
While high `seq_recovery` is good.

`score` --> arithmetic mean of the losses masked for design
`global_score` --> arithmetic mean of all losses (design+native)
